.devcontainer/devcontainer.json

```diff
@@ -1,12 +1,11 @@
 {
 	"name": "Go Build Environment",
 	"dockerFile": "../Dockerfile",
-	"forwardPorts": [8080],
+	"appPort": [8080],
```

Comment:
With Windows WSL, I needed to change forwardPorts to appPort. Tested also on MacBook with not issues.